client/state/preview/reducer.js

```diff
@@ -0,0 +1,69 @@
+/**
+ * External dependencies
+ */
+import assign from 'lodash/assign';
```

Comment:
It isn't a prototype method :) Ref: https://github.com/Automattic/wp-calypso/issues/2360#issuecomment-171296212